client/app/telecom/telephony/line/management/convert/telecom-telephony-line-management-convert.controller.js

```diff
@@ -57,5 +57,70 @@ angular.module("managerApp").controller("TelecomTelephonyLineConvertCtrl", funct
         });
     };
 
+    /* ===========================
+    =            BULK            =
+    ============================ */
+
+    self.bulkDatas = {
+        infos: {
+            name: "convertToNumber",
+            actions: [{
+                name: "convertToNumber",
+                route: "/telephony/{billingAccount}/line/{serviceName}/convertToNumber",
+                method: "POST",
+                params: null
+            }]
+        }
+    };
+
+    self.filterServices = function (services) {
+        var filteredServices = _.filter(services, function (service) {
+            return !_.some(service.offers, _.method("includes", "individual"));
+        });
+
+        return _.filter(filteredServices, function (service) {
+            return ["sip", "mgcp", "voicefax"].indexOf(service.featureType) > -1;
```

Comment:
Is voicefax featureType is managed by `/telephony/{billingAccount}/line/{serviceName}/convertToNumber` API ?

Reply:
I've seen with functionnal people, and yes voicefax is not available to line-to-number conversion. The conversion feature should not be available in management page of current line if feature type is voicefax